### ISSUE TITLE
Reset Customizer settings when a block theme is enabled

### ISSUE THREAD

**Issue by gigitux** [woothemes/woocommerce#42605]
With https://github.com/woocommerce/woocommerce/pull/34022, the Customizer is not accessible when a block theme is enabled, but at this stage, we don't reset the settings. 
So, for example, if with the Customizer, the user changes the dimension of the [Product Image thumbnail](https://woocommerce.com/document/woocommerce-customizer/#resize-and-crop-thumbnail-images) and after that enables a block theme, this customization is still active.

This could confuse the user.

**Comment by danieldudzic** [comment]
@gigitux What if someone activates a block theme, just to test but wants to go back, and is surprised they lost those settings?

**Comment by gigitux** [comment]
@danieldudzic yep, you are right! We should think well about all the cases. In any case, I opened this issue only for tracking reasons, we will not work on this, at least for now.
